packages/nextra-theme-docs/src/components/search.tsx

```diff
@@ -103,6 +105,9 @@ export function Search({
 
   const handleKeyDown = useCallback(
     <T,>(e: KeyboardEvent<T>) => {
+      // skip the character selection process in the IME for CJK language users.
+      if (e.nativeEvent.isComposing) return
+
```

Comment:
such as this:

https://github.com/user-attachments/assets/7de28dec-4908-448c-ae4e-86d401118c43